src/ruby/yast/wfm.rb

```diff
@@ -201,7 +201,7 @@ def self.call_builtin_wrapper(*args)
     # @param [Exception] e the caught exception
     # @return [String] human readable exception description
     private_class_method def self.internal_error_msg(e)
-      msg = "Internal error. Please report a bug report with logs.\n"
+      msg = "Internal error. Please report a bug report with logs from save_y2logs script.\n"
```

Comment:
what about adding "Run save_y2logs to get complete logs." ? otherwise it can get idea that this script do such reporting, which is not true.